sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantMessageModelAnnexTest.java

```diff
@@ -332,7 +333,12 @@ void testRequirementR5024() throws NoTestData, IOException {
     @TestDescription("For every DescriptionModificationReport received from the DUT, and for all parent-child"
             + " relationships between the elements contained in the report, checks that the reportPart containing"
             + " the parent comes before the reportPart containing the child.")
-    @RequirePrecondition(simplePreconditions = ConditionalPreconditions.DescriptionChangedPrecondition.class)
+    // NOTE: for this test case to succeed, the device should support descriptor updates or the list of removable
```

Comment:
Please remove this note. Chances are low that users will read it here. The readme provides information which manipulations have to be supported for this test.

Reply:
Moved the note into the message of the NoTestData Exception.